_source/user-guide/distributed-tracing/k8s-deployment.md

```diff
@@ -64,19 +236,24 @@ spec:
             port: 14269
         env:
         - name: ACCOUNT_TOKEN
-          value: <<ACCOUNT_TOKEN>> # Replace with the Tracing account token from Logz.io in Manage accounts > Distributed Tracing
```

Comment:
@yyyogev 
How does fetching the account token work now? 
Does the code get it automatically, without user intervention?

Reply:
from step 1